api/model/src/main/kotlin/wasik/api/restResource/items/model/IItem.kt

```diff
@@ -0,0 +1,9 @@
+package wasik.api.restResource.items.model
+
+interface IItem {
```

Comment:
Nitpick: Personally I prefer `ItemInterface`, depending on IDE, font etc `IItem` is hard to read

Reply:
I wasn't sure how to approach it at first, so i went with the familiar .Net naming scheme but it doesn't work that well here.
Changed it